cypress/integration/search-spec.js

```diff
@@ -46,10 +46,10 @@ describe('Article search', () => {
       .children().first().next().should('have.class', 'img')
   })
 
-  it('should show empty result found when search for "adf23cv1"', () => {
-    searchPage.search('adf23cv1')
-    searchPage.getEmptyContent().should('have.text', enJson['no-result-found'])
-  })
+  // it('should show empty result found when search for "adf23cv1"', () => {
```

Comment:
@jpita any idea why this test is failing? 

FYI: this is just a draft PR

Reply:
no tests are failing in my local.
did u remove the test?

Reply:
oh I see u commented the test.
next time just leave it failing in the CI until we find a cause, lost almost half an hour trying to understand which test u were talking about 😄 
so, ` searchPage.search('adf23cv1')` inserts that string in the search box, since now `1` goes to the language settings, it opens the settings...

I don't want to say "I told u so", but I did 😂

Reply:
Uhh that's a nasty bug 😄, like a UX bug even? Should probably make us reconsider. 

I'd have to look into it, maybe there's a way to only have "1" trigger the language change if user is not typing in the search bar? No idea how that would even work. But even then what about the case of a user trying to search "1" in search bar? 🤔

Reply:
anway, you can check `inputRef.current` whether is focus, similar to the center key example

```
center: current.type === 'DIV' ? i18n.i18n('centerkey-select') : ''
```

the design is changed and 1 is removed